docs/program_syntax.md

```diff
@@ -0,0 +1,69 @@
+# Program syntax
```

Comment:
@melver could you please proof read this for English? You are also better equipped to say what's non-obvious/hard to understand here.

Reply:
See below.


I think I mentioned this before, but there are good arguments for also formatting text-paragraphs at 80 cols, with the main one being the text remains readable in plain-text without editor hacks and doesn't force consumption in a web-browser. As-is, when opening these files in an editor, it will fill the whole width of the window (or screen) and many editors don't even auto-line break at their window width -- meaning it's hard to read in plain text (humans don't like reading long lines: https://baymard.com/blog/line-length-readability ).

I can't make that decision, so I'll leave it to you if this is what we want to start enforcing across syzkaller.

Reply:
We just need to find a markdown formatter that can enforce line length. Surprisingly the last time I checked I did not find any.

Reply:
Reformatted this markdown file at 80 cols (except code blocks).